en/FlyView/FlyView.md

```diff
@@ -1,79 +1,310 @@
 # Fly View
 
-![Fly View](../../assets/quickstart/ConnectedVehicle.jpg)
+The Fly View is used to command and monitor the vehicle when flying.
 
-The *Fly View* is the main view you will use while flying your vehicle.
-You can switch between a map view and a video view (if available).
+You can use it to:
```

Comment:
FYI, note high level description of what you can do, and forward links to those sections